File: src/B_1284/B_1284.java
package B_1284;

import java.io.*;

public class B_1284 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();
        int n = Integer.parseInt(br.readLine());
        while(n!=0){
            sb.append(addr(n)).append('\n');
            n = Integer.parseInt(br.readLine());
        }
        System.out.println(sb);

    }
    static int addr(int x){
        int res = 1;
        while(x>0){
            int n = x%10;
            x/=10;
            if(n==1){
                res += 3;
            }else if(n==0){
                res += 5;
            }else{
                res += 4;
            }
        }
        return res;
    }

}

File: src/B_1284/B_1284.kt
package B_1284

import java.io.*

fun main(args: Array<String>) {
    val br = BufferedReader(InputStreamReader(System.`in`))
    val sb = StringBuilder()
    var n = br.readLine().toInt()
    while (n != 0) {
        sb.append(addr(n)).append('\n')
        n = br.readLine().toInt()
    }
    println(sb)
}

fun addr(x: Int): Int {
    var x = x
    var res = 1
    while (x > 0) {
        val n = x % 10
        x /= 10
        res += when(n){
            1 -> 3
            0 -> 5

            else -> 4
        }
    }
    return res
}

